vllm/v1/attention/backends/flash_attn.py

```diff
@@ -82,8 +82,10 @@ def __init__(
         if alibi_slopes is not None:
             alibi_slopes = torch.tensor(alibi_slopes, dtype=torch.float32)
         self.alibi_slopes = alibi_slopes
-        self.sliding_window = ((sliding_window, sliding_window)
-                               if sliding_window is not None else (-1, -1))
+        if sliding_window is None:
+            self.sliding_window = (-1, -1)
+        else:
+            self.sliding_window = ((sliding_window - 1, 0))
```

Comment:
Can you remove redundant brackets to be  `self.sliding_window = (sliding_window - 1, 0)`  ?

Reply:
Good catch! Fixed.